### ISSUE TITLE
Tensorflow's doc - 2 Conv+pooling benchmark update

### ISSUE THREAD

**Issue by gustavovelascoh** [zalandoresearch/fashion-mnist#129]
I have tested the code related in the benchmark for [Tensorflow's doc](https://github.com/zalandoresearch/fashion-mnist/blob/master/benchmark/convnet.py) and got accuracy of  0.9905. Below is the final output of running the code:

```
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 198000: /tmp/mnist_convnet_model/model.ckpt-198000
{'global_step': 198000, 'accuracy': 0.9905, 'loss': 0.031072732}
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Create CheckpointSaverHook.
INFO:tensorflow:Graph was finalized.
2018-09-14 13:16:34.373255: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2018-09-14 13:16:34.373300: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-09-14 13:16:34.373306: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2018-09-14 13:16:34.373310: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2018-09-14 13:16:34.373389: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 3020 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1050 Ti, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO:tensorflow:Restoring parameters from /tmp/mnist_convnet_model/model.ckpt-198000
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 198000 into /tmp/mnist_convnet_model/model.ckpt.
INFO:tensorflow:loss = 0.009688791, step = 198000
INFO:tensorflow:global_step/sec: 31.2563
INFO:tensorflow:loss = 0.014211949, step = 198100 (3.200 sec)
INFO:tensorflow:global_step/sec: 31.557
INFO:tensorflow:loss = 0.01864367, step = 198200 (3.169 sec)
INFO:tensorflow:global_step/sec: 31.5311
INFO:tensorflow:loss = 0.022106394, step = 198300 (3.172 sec)
INFO:tensorflow:global_step/sec: 31.6026
INFO:tensorflow:loss = 0.01054849, step = 198400 (3.164 sec)
INFO:tensorflow:global_step/sec: 30.5505
INFO:tensorflow:loss = 0.022961156, step = 198500 (3.273 sec)
INFO:tensorflow:global_step/sec: 30.0051
INFO:tensorflow:loss = 0.030964922, step = 198600 (3.333 sec)
INFO:tensorflow:global_step/sec: 30.3238
INFO:tensorflow:loss = 0.032633875, step = 198700 (3.298 sec)
INFO:tensorflow:global_step/sec: 30.6906
INFO:tensorflow:loss = 0.03055433, step = 198800 (3.258 sec)
INFO:tensorflow:global_step/sec: 29.7538
INFO:tensorflow:loss = 0.023177711, step = 198900 (3.361 sec)
INFO:tensorflow:global_step/sec: 30.3007
INFO:tensorflow:loss = 0.01991336, step = 199000 (3.300 sec)
INFO:tensorflow:global_step/sec: 29.4653
INFO:tensorflow:loss = 0.018575167, step = 199100 (3.394 sec)
INFO:tensorflow:global_step/sec: 29.3697
INFO:tensorflow:loss = 0.013256683, step = 199200 (3.405 sec)
INFO:tensorflow:global_step/sec: 29.7985
INFO:tensorflow:loss = 0.009808191, step = 199300 (3.356 sec)
INFO:tensorflow:global_step/sec: 30.6575
INFO:tensorflow:loss = 0.02743866, step = 199400 (3.262 sec)
INFO:tensorflow:global_step/sec: 29.4652
INFO:tensorflow:loss = 0.010928765, step = 199500 (3.394 sec)
INFO:tensorflow:global_step/sec: 28.6025
INFO:tensorflow:loss = 0.01802608, step = 199600 (3.496 sec)
INFO:tensorflow:global_step/sec: 29.2092
INFO:tensorflow:loss = 0.02041433, step = 199700 (3.424 sec)
INFO:tensorflow:global_step/sec: 28.7581
INFO:tensorflow:loss = 0.027336344, step = 199800 (3.478 sec)
INFO:tensorflow:global_step/sec: 30.0796
INFO:tensorflow:loss = 0.0358267, step = 199900 (3.324 sec)
INFO:tensorflow:Saving checkpoints for 200000 into /tmp/mnist_convnet_model/model.ckpt.
INFO:tensorflow:Loss for final step: 0.031748984.
INFO:tensorflow:Calling model_fn.
INFO:tensorflow:Done calling model_fn.
INFO:tensorflow:Starting evaluation at 2018-09-14-12:17:41
INFO:tensorflow:Graph was finalized.
2018-09-14 13:17:41.088195: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1484] Adding visible gpu devices: 0
2018-09-14 13:17:41.088222: I tensorflow/core/common_runtime/gpu/gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-09-14 13:17:41.088227: I tensorflow/core/common_runtime/gpu/gpu_device.cc:971]      0 
2018-09-14 13:17:41.088230: I tensorflow/core/common_runtime/gpu/gpu_device.cc:984] 0:   N 
2018-09-14 13:17:41.088329: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 3020 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1050 Ti, pci bus id: 0000:01:00.0, compute capability: 6.1)
INFO:tensorflow:Restoring parameters from /tmp/mnist_convnet_model/model.ckpt-200000
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2018-09-14-12:17:41
INFO:tensorflow:Saving dict for global step 200000: accuracy = 0.9905, global_step = 200000, loss = 0.03111329
INFO:tensorflow:Saving 'checkpoint_path' summary for global step 200000: /tmp/mnist_convnet_model/model.ckpt-200000
{'global_step': 200000, 'accuracy': 0.9905, 'loss': 0.03111329}
```
Should be updated?

**Comment by hanxiao** [comment]
One common mistake is that people thought they are working with Fashion-MNIST, but they are actually working with MNIST. This mistake can be found repeatedly in #110 #47 #36 #119  etc.

Our [benchmark table](https://github.com/zalandoresearch/fashion-mnist/issues/36) (contributed by many people) also suggests that it's impossible for simple CNN to get 99% test accuracy.

I'm sorry to say that, but it's highly likely due to some bug in your code. Please check your code again and again.

Here are two ways to check whether you are working with the correct dataset.

1. Check the MD5 of your dataset. And compare it with our [MD5 in the README.md](https://github.com/zalandoresearch/fashion-mnist#get-the-data). Details can be found [in this thread](https://github.com/zalandoresearch/fashion-mnist#get-the-data).
2. Plot your dataset, [as suggested here](https://github.com/mpekalski/zalando/blob/master/problem.ipynb)

Good luck!

**Comment by gustavovelascoh** [comment]
Thanks for your reply.
